references/obj_detection/train_pytorch.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import os
+
+os.environ['USE_TORCH'] = '1'
+
+import time
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+
+from doctr.datasets import DocArtefacts
+from model import faster_model
+from utilities import *
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
+def fit_one_epoch(model_, images, targets):
+    x = images.to(device)
+    height, width = x.shape[0], x.shape[1]
+    for i in targets["boxes"]:
+        i[0] = round(i[0] * width)
+        i[2] = round(i[2] * width)
+        i[1] = round(i[1] * height)
+        i[2] = round(i[3] * height)
```

Comment:
targets are list of dict, I don't think this will work 

```python
for idx in range(x.shape[0]):
    target[idx]['boxes'][:, 0::2] = (target[idx]['boxes'][:, 0::2] * width).round()
    target[idx]['boxes'][:, 1::2] = (target[idx]['boxes'][:, 1::2] * height).round()
```

Reply:
Thanks:) Have made the changes